PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/org/jetbrains/plugins/azure/functions/run/localsettings/FunctionLocalSettings.kt

```diff
@@ -39,5 +39,5 @@ data class FunctionHostModel(val localHttpPort: Int?, val cors: String?, val cor
 
 enum class FunctionsWorkerRuntime(val value: String) {
     DotNetDefault("dotnet"),
```

Comment:
Could change this as well, for consistency.

Reply:
Let's change